org.jacoco.core.test/src/org/jacoco/core/test/TargetLoader.java

```diff
@@ -52,17 +52,24 @@ public TargetLoader() {
 	}
 
```

Comment:
Should be consistent now. But kept the classes map otherwise we need to assert that only one class is inserted.